docs/topics/crud-mission-intro-paragraph.adoc

```diff
@@ -1 +1 @@
-The CRUD application Booster expands on the {name-mission-http-api} Booster to provide a basic example of performing _create_, _read_, _update_ and _delete_ (_CRUD_) operations on a PostgreSQL database using a simple HTTP API. _CRUD_ operations are the four basic functions of persistent storage, widely used when developing HTTP API dealing with a database.
\ No newline at end of file
+The Relational Database Backend application Booster expands on the {name-mission-http-api} Booster to provide a basic example of performing _create_, _read_, _update_ and _delete_ (_CRUD_) operations on a PostgreSQL database using a simple HTTP API. _CRUD_ operations are the four basic functions of persistent storage, widely used when developing HTTP API dealing with a database.
```

Comment:
"Relational Database Backend application" looks weird. It's a CRUD application, sure, but again, it's not a "relational database backend" for the frontend. Maybe just "The Relational Database Backend Booster expands on ..."?